apps/api-gateway/src/issuance/issuance.controller.ts

```diff
@@ -432,7 +445,7 @@ async downloadBulkIssuanceCSVTemplate(
         type: fileDetails?.['type']
       };
     }
-      const bulkIssuanceDetails = await this.issueCredentialService.issueBulkCredential(requestId, orgId, clientDetails, reqPayload);
+      const bulkIssuanceDetails = await this.issueCredentialService.issueBulkCredential(requestId, orgId, clientDetails, reqPayload, isValidateSchema);
```

Comment:
```suggestion
      const bulkIssuanceDetails = await this.issueCredentialService.issueBulkCredential(requestId, orgId, clientDetails, reqPayload, isValidateSchema);
```
We can pass single object insted of passing multiple parameters

Reply:
Yes, resolved.